includes/core/classes/class-event-setup.php

```diff
@@ -137,13 +139,38 @@ public function register_post_type(): void {
 					'custom-fields',
 				),
 				'menu_icon'     => 'dashicons-nametag',
+				'has_archive'   => true,
 				'rewrite'       => array(
-					'slug' => _x( 'event', 'Post Type Slug', 'gatherpress' ),
+					'slug'       => $rewrite_slug,
+					'with_front' => false,
 				),
 			)
 		);
 	}
 
+	/**
+	 * Returns the post type slug localized for the site langugage and sanitized as URL part.
+	 *
+	 * Do not use this directly, use get_value( 'general', 'urls', 'events' ) instead.
+	 *
+	 * This method switches to the sites default language and gets the translation of 'events' for the loaded locale.
+	 * After that, the method sanitizes the string to be safely used within an URL,
+	 * by removing accents, replacing special characters and replacing whitespace with dashes.
+	 *
+	 * @since 0.31.0
```

Comment:
Looks to be still 0.31.0. Let's make it 1.0.0 just to be consistent. I think we're too far in to figure out which methods/classes came at which version so just sticking to 1.0.0 and going from there is probably most doable now.

Reply:
Oops….